mw-webapp/src/styles/_variables.scss

```diff
@@ -34,8 +34,10 @@ $gapBig: 20px;
 $gapExtraMedium: 16px;
 $gapMedium: 8px;
 $gapSmall: 5px;
+$gapLarge: 36px;
 
 // Widths and heights
+$maxWidth: 1370px;
```

Comment:
why it is not near all width?

why such number? looks unconsistent

Reply:
according to the design layout in figma we have container size 1560px

Reply:
changed 1560px